pkg/cvo/sync_worker.go

```diff
@@ -674,6 +686,8 @@ func (w *statusWrapper) ReportPayload(payloadStatus LoadPayloadStatus) {
 
 // Report reports payload application status. It does not overwrite payload load status and capabilities status
 // since payload application does not update these statuses they could therefore be out-of-date.
+//
+// SyncWorker w.w must not be locked before ReportPayload is called.
```

Comment:
Copy paste error, s/ReportPayload/Report/

Reply:
will fix

Reply:
fixed, also changed the godocs according to our Slack convo